app/routes/pages/components/PageDetail.js

```diff
@@ -82,41 +99,134 @@ export const GroupRenderer = ({ page }: { page: Object }) => {
   );
 };
 
-function PageDetail<T: Object>({
-  selectedPage,
-  selectedPageInfo,
-  pageHierarchy,
-  PageRenderer,
-  currentUrl
-}: Props<T>) {
-  if (!selectedPage) {
-    return <LoadingIndicator loading />;
+class Sidebar extends Component {
+  render() {
+    const {
+      categorySelected,
+      currentUrl,
+      pageHierarchy,
+      isOpen,
+      handleClose
+    } = this.props;
+    const pictureLabel = 'Listingløpet 1985';
+
+    return (
+      <div
+        className={isOpen ? styles.sidebarWrapper : undefined}
+        onClick={handleClose}
+      >
+        <div
+          className={cx(styles.side, isOpen && styles.isOpen)}
+          onClick={event => {
+            // Ask someone about this (or just use gogle) <3
```

Comment:
No comprendo :thinking: 

I understand what `stopPropagation` does, but not why it is necessary since the onClickHandler doesn't do anything else :man_shrugging:

Reply:
If you don't have it, you cannot use the mobile menu. Bcause of the overlay in the background that closes the menu when pressed, if you remove `stopPropagation`, every time you attempt to click anywhere in the menu will close the menu.